fg2: <main> function()
  block 0: [entry]
    node 0: constant[undefined,v1] (test-resources/src/flowgraphbuilder/flowgraph_builder0075.js)
    ->[block 1]
  block 1:
    node 1: constant["tajs-host-env:string-replace-model.js",v2] (host-environment-sources-loader)
    node 2: constant[true,v3] (host-environment-sources-loader)
    ->[block 2]
  block 2:
    node 3: call[TAJS_load,v2,v3,v4] (host-environment-sources-loader)
    ->[block 3]
  block 3:
    node 4: call[-,v4,-] (host-environment-sources-loader)
    ->[block 4]
  block 4:
    node 5: constant[3.0,-]* (test-resources/src/flowgraphbuilder/flowgraph_builder0075.js:1:7)
    ->[block 7] ~>[block 5]
  block 5:
    node 6: catch[e,v6] (test-resources/src/flowgraphbuilder/flowgraph_builder0075.js:1:11)
    node 7: begin-with[v6]* (test-resources/src/flowgraphbuilder/flowgraph_builder0075.js:1:11)
    ->[block 6] ~>[block 9]
  block 6:
    node 8: end-with* (test-resources/src/flowgraphbuilder/flowgraph_builder0075.js:1:11)
    ->[block 7] ~>[block 9]
  block 7:
    node 9: constant[4.0,-]* (test-resources/src/flowgraphbuilder/flowgraph_builder0075.js:1:35)
    ->[block 10]
  block 8:
    node 10: end-with* (test-resources/src/flowgraphbuilder/flowgraph_builder0075.js:1:11)
    ->[block 9]
  block 9:
    node 11: catch[v5] (test-resources/src/flowgraphbuilder/flowgraph_builder0075.js:1:1)
    node 12(~9): constant[4.0,-]* (test-resources/src/flowgraphbuilder/flowgraph_builder0075.js:1:35)
    node 13: throw[v5]* (test-resources/src/flowgraphbuilder/flowgraph_builder0075.js:1:1)
    ->[block 11]
  block 10: [exit-ordinary]
    node 14: return[v1] (test-resources/src/flowgraphbuilder/flowgraph_builder0075.js)
    ->[]
  block 11: [exit-exceptional]
    node 15: exceptional-return (test-resources/src/flowgraphbuilder/flowgraph_builder0075.js)
    ->[]

test-resources/src/flowgraphbuilder/flowgraph_builder0075.js: [definite] Exceptional program exit is unreachable
test-resources/src/flowgraphbuilder/flowgraph_builder0075.js:1:11: [definite] Unreachable code
Soundness testing succeeded for 1 checks (with 0 expected failures)
